src/server/qgsserverrequest.cpp

```diff
@@ -28,18 +29,33 @@ QgsServerRequest::QgsServerRequest( const QString &url, Method method, const Hea
 QgsServerRequest::QgsServerRequest( const QUrl &url, Method method, const Headers &headers )
   : mUrl( url )
   , mOriginalUrl( url )
+  , mBaseUrl( url )
   , mMethod( method )
   , mHeaders( headers )
-{
+  , mRequestHeaderConv()
+{
+  mRequestHeaderConv.insert( HOST, "Host" );
```

Comment:
```suggestion
  mRequestHeaderConv.insert( HOST, QStringLiteral( "Host" ) );
```
... and all other as well